git_machete/client.py

```diff
@@ -1040,11 +1040,10 @@ def diff(self, branch: Optional[str]) -> None:
             [fp] + \
             ([f"refs/heads/{branch}"] if branch else []) + \
             ["--"]
-        self.__git.run_git("diff", *params)
+        self.__git.run_diff(*params)
 
     def log(self, branch: str) -> None:
-        self.__git.run_git(
-            "log",
+        self.__git.run_log(
```

Comment:
Distinction b/w `run_log` and `get_log` is very needed indeed, the former **displays** the log (probably using a pager like `less` for scrolling), while the latter serves for just retrieving the output